packages/ui/src/utils.ts

```diff
@@ -78,6 +78,21 @@ export function floorFractionDigits (n: number | string, amount: number): number
   return Number(Number(n).toFixed(amount))
 }
 
+/**
+ * @public
+ */
+export function humanReadableFileSize (size: number, base: 2 | 10 = 10, fractionDigits: number = 2): string {
+  const units =
+    base === 10
+      ? ['B', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB']
```

Comment:
Internationalisation?

Reply:
unit names are international and don't require translation